test/framework/integration_framework/integration_test_framework.hpp

```diff
@@ -126,6 +131,36 @@ namespace integration_framework {
     // check validation function
     return *this;
   }
+
+  template <typename Lambda>
+  IntegrationTestFramework &IntegrationTestFramework::sendQuery(
+      const iroha::model::Query &qry, Lambda validation) {
+    using iroha::operator|;
+    log_->info("send query");
+    // serialize without calling destructor on passed reference
+    auto pb_qry = iroha::model::converters::PbQueryFactory().serialize(
+        std::shared_ptr<const iroha::model::Query>(&qry, [](auto) {}));
+    pb_qry | [&](const auto &pb_qry) {
+      // send
+      iroha::protocol::QueryResponse pb_response;
+      iroha_instance_->getIrohaInstance()->getQueryService()->FindAsync(
+          pb_qry, pb_response);
+      // deserialize
+      auto response =
+          iroha::model::converters::PbQueryResponseFactory().deserialize(
+              pb_response);
+      // check validation function
+      response | [&](const auto &p) { validation(*p); };
+      if (not response) {
+        log_->info("query response deserialization failed");
+      }
+    };
+    if (not pb_qry) {
+      log_->info("query serialization failed");
+    }
+    return *this;
+  }
+
   template <typename Lambda>
   IntegrationTestFramework &IntegrationTestFramework::checkBlock(
       Lambda validation) {
```

Comment:
this fn is also not covered

Reply:
Which function? If it was not added in this pull request, then it should be covered in a separate task.